tests/e2e/test_main.py

```diff
@@ -932,6 +934,9 @@ def _f(
                         "AZURE_SAS_TOKEN=secret:AZURE_SAS_TOKEN",
                     ]
                 )
+            elif bucket.startswith("https://"):
```

Comment:
so we test only https? or is there an error that the branch `elif bucket.startswith("http://")` is missing?